test/test_jit.py

```diff
@@ -17486,7 +17517,7 @@ def test_docs(self):
     add_nn_module_test(**test)
 
 if __name__ == '__main__':
-    run_tests()
+    # run_tests()
```

Comment:
did you mean to include this?

Reply:
It's for quick debug. Just added it back. Thanks!